pkg/manager/impl/node_execution_manager.go

```diff
@@ -437,9 +437,12 @@ func (m *NodeExecutionManager) GetNodeExecutionData(
 		logger.Debugf(ctx, "failed to transform node execution model [%+v] when fetching data: %v", request.Id, err)
 		return nil, err
 	}
-	signedInputsURLBlob, err := m.urlData.Get(ctx, nodeExecution.InputUri)
-	if err != nil {
-		return nil, err
+	signedInputsURLBlob := admin.UrlBlob{}
+	if nodeExecution.InputUri != "" {
+		signedInputsURLBlob, err = m.urlData.Get(ctx, nodeExecution.InputUri)
```

Comment:
nit:
```suggestion
	signedInputsURLBlob := admin.UrlBlob{}
	if len(nodeExecution.InputUri) != 0 {
		signedInputsURLBlob, err = m.urlData.Get(ctx, nodeExecution.InputUri)
```

Reply:
Thanks . Will take this as an immediate followup in the scheduler pr.